troubleshooting/known_issues/missing_binaries_in_aarch64_images.md

```diff
@@ -0,0 +1,120 @@
+    Aug 16 12:00:23 nid001048 (serm[2953]: cxi_rh@cxi0.service: Executable /usr/bin/fusermount missing, skipping: No such file or directory
+    Aug 16 12:00:23 nid001048 cxi_rh[2974]: /usr/bin/cxi_rh: error while loading shared libraries: libfuse.so.2: cannot open shared object file: No such file or directory
+    Aug 16 12:00:23 nid001048 systemd[1]: cxi_rh@cxi0.service: Main process exited, code=exited, status=127/n/a
+    Aug 16 12:00:23 nid001048 systemd[1]: cxi_rh@cxi0.service: Failed with result 'exit-code'.
+    Aug 16 12:00:23 nid001048 systemd[1]: Failed to start CXI Retry Handler on cxi0.
+    Oct 19 20:09:03 nid001048 systemd[1]: Starting CXI Retry Handler on cxi0...
+    Oct 19 20:09:03 nid001048 (serm[5475]: cxi_rh@cxi0.service: Executable /usr/bin/fusermount missing, skipping: No such file or directory
+    Oct 19 20:09:03 nid001048 cxi_rh[5477]: /usr/bin/cxi_rh: error while loading shared libraries: libfuse.so.2: cannot open shared object file: No such file or directory
+    Oct 19 20:09:03 nid001048 systemd[1]: cxi_rh@cxi0.service: Main process exited, code=exited, status=127/n/a
+    Oct 19 20:09:03 nid001048 systemd[1]: cxi_rh@cxi0.service: Failed with result 'exit-code'.
+    Oct 19 20:09:03 nid001048 systemd[1]: Failed to start CXI Retry Handler on cxi0.
+    ```
+
+    From this it was observed that the `libfuse.so.2` library was missing. To resolve this, the missing libraries
+    were added explicitly to the ansible playbook where the package was installed.
+
+1. Missing dependency on `liblnetconfig.so.4`.
+
+    In this case the missing shared object file was reported directly during the dracut phase of the boot:
+
+    ```text
+    131.772352] dracut-initqueue[3952]: cps: All requested interfaces are UP, proceeding.
+    [  131.958603] dracut-initqueue[4013]: 4 blocks
+    [  132.803820] dracut-pre-mount[4067]: LNet: loaded lnet module.
+    [  132.840080] dracut-pre-mount[4077]: lnetctl: error while loading shared libraries: liblnetconfig.so.4: cannot open shared object file: No such file or directory
+    [  132.840141] dracut-pre-mount[4067]: LNet: Error calling 'lnetctl lnet configure'.
+    [  132.840181] dracut-pre-mount[4065]: DVS: ERROR: lnet-load.sh failed.
+    [  132.840195] dracut-pre-mount[4063]: Warning: ERROR: dvs-setup.sh failed; dropping to debug.
+    [  132.840210] dracut-pre-mount[4058]: Warning: Unable to prepare squashfs file /tmp/cps/rootfs, dropping to debug.
+
+    Generating "/run/initramfs/rdsosreport.txt"
+
+    Press Enter for maintenance
+    (or press Control-D to continue): 
+    ```
+
+    This case directly reported the missing `liblnetconfig.so.4` file, so any of the below workaround steps
+    could be taken to resolve the issue.
+
+## Workarounds
+
+There are a couple of ways to work around this issue once it has been identified.
+
+### Build or Customize the Image on a Remote Node (Only in CSM 1.5.2 or later)
```

Comment:
Fixed the casing, and also removed the 1.5.2 reference that isn't needed in the CSM 1.6 docs

```suggestion
### Build or customize the image on a remote node
```